include/boost/thread/detail/thread.hpp

```diff
@@ -718,7 +713,7 @@ namespace boost
     };
 
 #ifdef BOOST_THREAD_PLATFORM_PTHREAD
-    thread::id thread::get_id() const BOOST_NOEXCEPT
+    inline thread::id thread::get_id() const BOOST_NOEXCEPT
```

Comment:
There is no need to add inline here. It is better to add it on the declarations AFAIK.

Reply:
According to this FAQ, it is better to place inline next to the definition instead of the declaration. Do you think this FAQ is correct, as far as you know? https://isocpp.org/wiki/faq/inline-functions#where-to-put-inline-keyword

Reply:
I was not aware of this. I agree that the code is more readable.
As it is in the isocpp.org site, lets say that it should be portable. I've no time to do search.